### ISSUE TITLE
How to Manually Update

### ISSUE THREAD

**Issue by espanafly** [portapps/brave-portable#134]
### **Guide to Manually update the portapps brave-portable package**
See that you have an in-browser notification about the need to update 
![Notice](https://user-images.githubusercontent.com/69840998/227748193-206497fe-7c03-4ed6-a2a4-108718a004db.jpg)

- Go and look for update on the website [Brave Browser Portable - Download - COMPUTER BILD](https://www.computerbild.de/download/Brave-Browser-Portable-31712155.html)
- Confirm there is a new build by looking at the "Spezifikationen" or "Specifications" section
- Download the compressed computerbild archive: Brave_Portable_x86_x64.zip
- Open the computerbild archive
- Inside the archive, open the x64 binary folder: "brave-v1.XX.XXX-win32-x64"
- Close your portapps Brave browser
- Open the "app" folder inside your portapps Brave portable (x64) folder
- Delete the stale-outdated content inside the "app" folder: two files and one folder
- Drag and drop the replacement (x64) files/folder from the computerbild archive

![UpdateNew](https://user-images.githubusercontent.com/69840998/227748702-ab613ed5-355e-49cb-9f3e-cb447f549cab.jpg)
Restart your manually updated portapps Brave browser

**== stuff to know ===**
_Track record_ 
I've used this method to update twice successfully. Replace the "app" folder content _only_! Leave everything else alone!
_Acknowledgements_
Thanks [CrazyMax](https://crazymax.dev/)
This guide depends on the portable package provided by computerbild.de. If they stop maintaining their portable package, then this guide becomes useless.

**Comment by greatwolf** [comment]
It's better to follow my procedure here: https://github.com/portapps/brave-portable/issues/129#issuecomment-1382631183

Since its download source is from brave.com itself rather than some unknown website.

**Comment by espanafly** [comment]
I agree. Your procedure is clear enough and the brave download source is better than mine.
The problem was that your post was a buried in a reply to a "closed as completed" issue and so not so easy to find. Not so easy for me to find anyway.
"[Computer Bild first published in 1996](https://en.wikipedia.org/wiki/Computer_Bild)" is the German equivalent to [PCMag first published from 1982.](https://en.wikipedia.org/wiki/PCMag)
If anyone is interested, I did a  hash compare between the brave developers files and the COMPUTER BILD files and they were identical.

I'm going to suggest (via pull request) that your procedure get incorporated into this projects readme.

> It's better to follow my procedure here: [#129 (comment)](https://github.com/portapps/brave-portable/issues/129#issuecomment-1382631183)
> 
> Since its download source is from brave.com itself rather than some unknown website.

**Comment by Chaython** [comment]
https://github.com/Chaython/Brave-Portable-Updater
